inputs_replace.py

```diff
@@ -0,0 +1,30 @@
+from mwrogue.esports_client import EsportsClient
+from mwrogue.auth_credentials import AuthCredentials
+from mwcleric.errors import RetriedLoginAndStillFailed
+import mwparserfromhell
+
+credentials = AuthCredentials(user_file="bot")
+site = EsportsClient("lol", credentials=credentials, max_retries=0)
+
+response = site.cargo_client.query(
+    tables="Teamnames=T, Teamnames__Inputs=I, Teamnames__Inputs=I2",
+    where="I._value LIKE '%,%' AND I2._value NOT LIKE '%,%'",
+    join_on="T._ID=I._rowID, I._rowID=I2._rowID",
+    fields="I._value=BadInput, I2._value=GoodInput, T.Link",
+    group_by="T._ID"
+)
+
+for item in response:
+    for page in site.client.pages[item["Link"]].backlinks():
+        page_text = page.text()
```

Comment:
nitpick: rename `page_text` to `original_text` because later on you're comparing the post-edit text to the original text, and that comparison is slightly confusing when both of them are named page_text / page_wikitext